editor/src/clj/editor/app_view.clj

```diff
@@ -1320,23 +1320,32 @@ If you do not specifically require different script states, consider changing th
     (console/pipe-log-stream-to-console! in)
     (PipedOutputStream. in)))
 
+(defn- build-html5! [project prefs web-server build-errors-view changes-view main-stage tool-tab-pane pre-bob-cmds]
+  (let [main-scene (.getScene ^Stage main-stage)
+        render-build-error! (make-render-build-error main-scene tool-tab-pane build-errors-view)
+        render-reload-progress! (make-render-task-progress :resource-sync)
+        render-save-progress! (make-render-task-progress :save-all)
+        render-build-progress! (make-render-task-progress :build)
+        task-cancelled? (make-task-cancelled-query :build)
+        bob-args (bob/build-html5-bob-options project prefs)
+        out (start-new-log-pipe!)]
+    (build-errors-view/clear-build-errors build-errors-view)
+    (disk/async-bob-build! render-reload-progress! render-save-progress! render-build-progress! out task-cancelled?
+                           render-build-error! (vec (concat pre-bob-cmds bob/build-html5-bob-commands)) bob-args project changes-view
+                           (fn [successful?]
+                             (when successful?
+                               (ui/open-url (format "http://localhost:%d%s/index.html" (http-server/port web-server) bob/html5-url-prefix)))
+                             (.close out)))))
+
+(handler/defhandler :rebuild-html5 :global
```

Comment:
new option that passes extra build commands

Reply:
Hmm, I think it would be clearer if instead of `pre-bob-commands` this function accepted `bob-commands`. Then we would have `bob/build-html5-bob-commands` and `bob/rebuild-html5-bob-commands`. I think it would be easier to follow.

Reply:
yes, this is much better! Thanks! Fixed!